ibis/backends/duckdb/tests/test_client.py

```diff
@@ -276,3 +276,12 @@ def test_invalid_connect(tmp_path):
     url = f"duckdb://{tmp_path}?read_only=invalid_value"
     with pytest.raises(ValueError):
         ibis.connect(url)
+
+
+@pytest.mark.xfail(
+    raises=(duckdb.InvalidInputException, duckdb.BinderException),
```

Comment:
Why are there two exceptions here?

Reply:
Depending on the python/duckdb version you get different errors, was running into both in CI. I wasn't able to actually figure out the exact pattern, I couldnt repro one of the errors locally.  You want a comment for this? Actually figure it out?

Reply:
OK I pushed with this xfail commented out so you can see [CI failures](https://github.com/ibis-project/ibis/actions/runs/8381568818?pr=8649):

- "duckdb + GeoSpatial *" tests raises BinderErrors
- the other duckdb tests raise InvalidInputExceptions

Reply:
@cpcloud gentle nudge here :)

Reply:
Sorry I missed this!

Totally fine to keep both exception types.